tools/prepare-source.sh

```diff
@@ -2,6 +2,14 @@
 
 set -eux
 
-go fmt ./...
-go mod tidy
-go mod vendor
+GO_VERSION=1.12.17
+GO_BINARY=$(go env GOPATH)/bin/go$GO_VERSION
+
+# get golang of version GO_VERSION
+go get golang.org/dl/go$GO_VERSION
+$GO_BINARY download
```

Comment:
Do you mind adding a link to this here: https://golang.org/doc/install#extra_versions, just to clarify this is an officially supported way to get at a different version of go.

Unless I'm the only one that stumbled across that and started researching :)

Reply:
I added the link, thanks for the feedback!